src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TodoItem } from '../TodoItem';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  isFilter: string | undefined;
+}
+
+export const TodoList: React.FC<Props> = ({ todos, setTodos, isFilter }) => {
+  let filteredTodos: Todo[] = [];
+
+  if (isFilter === 'Completed') {
+    filteredTodos = todos.filter(todo => todo.completed);
+  } else if (isFilter === 'Active') {
+    filteredTodos = todos.filter(todo => !todo.completed);
+  } else {
+    filteredTodos = todos;
+  }
```

Comment:
use switch case here.
move this logic into app component.
your todo list should only render todos